### ISSUE TITLE
PDF label printing 

### ISSUE THREAD

**Issue by ManiaChamp** [electricimp/QL720NW#8]
Do you have any sample for PDF label printing?

**Comment by betzrhodes** [comment]
We don't, the current driver only supports text and bar code labels.

**Comment by ManiaChamp** [comment]
Can we put this as a feature request?

**Comment by hfiennes** [comment]
No, rendering PDFs is beyond the capabilities of an imp or any label printer I’m aware of. That requires megabytes of ram.

You can render PDF to bitmap on a server and send the bitmap stream to the printer via an imp but that’s not what this driver does.
